initializers/object_permissions.yml

```diff
@@ -0,0 +1,22 @@
+#- name: all.ro
+#  description: 'Read Only for All Objects'
+#  enabled: true
+#  # object_types: all
+#  groups:
+#    - applications
+#    - readers
+#  actions:
+#    - view
+#- name: all.rw
```

Comment:
Are these arbitrary names? I.e. does the user define them?

Reply:
Ah I commented this in the `object_permissions.py` file, but forgot to mention in the PR.  I was having trouble with adding the individual object_types.  I need to take the list in said dictionary for key `object_types` and send to the Django ORM (nbshell) to get the proper model items to associate to the object permission.  I hit a road block, so I hacked it to set all for now.  The example just documents that, but again I should have added some more comments. 

Would you mind weighing in on the commented out lines to see what I am missing when trying to grab the possible object_types based on the list `object_types` in the yaml?

**object_permissions.py:**
```python
    # Need to try to pass a list of model_name and app_label for more than the current ALL
    # object_types = ContentType.objects.filter(app_label__in=params.pop("object_types"))
    # object_permission.object_types.set(ContentType.objects.filter(app_label__in=params.pop("object_types")))
    object_permission.object_types.set(ContentType.objects.all())
    object_permission.save()
```